aptos-move/aptos-gas/src/move_stdlib.rs

```diff
@@ -30,4 +30,4 @@ crate::natives::define_gas_parameters_for_natives!(GasParameters, "move_stdlib",
     // TODO(Gas): rename these in the move repo
     [test_only .unit_test.create_signers_for_testing.base_cost, "unit_test.create_signers_for_testing.base", 1],
     [test_only .unit_test.create_signers_for_testing.unit_cost, "unit_test.create_signers_for_testing.unit", 1]
-], allow_unmapped = 1 /* bcs */ + 2 /* hash */ + 8 /* vector */);
+], allow_unmapped = 1 /* bcs */ + 2 /* hash */ + 8 /* vector */ + 2 /* type_name */);
```

Comment:
ah, to be clear we aren't importing this into our representation of stdlib and this is only used for test assertion?

other than that, seems pretty straight forward

Reply:
Yes type_name not reachable from our lib but I wanted to cherry pick the PR with it because that one also introduces canonical account address strings. Victor need to check whether my test fix here sufficient

Reply:
@vgao1996